<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>560</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculadora</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Calculadora.ico</normaloff>Calculadora.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">	margin: 5px;
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 50px;	
    background-color: black ;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	margin: 5px;
    background-color: rgb(85, 170, 255;
	background-color:);
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>290</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	margin: 5px;
    background-color: rgb(85, 170, 255;
	background-color:);
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn6">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>290</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	margin: 5px;
    background-color: rgb(85, 170, 255;
	background-color:);
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMenos">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>290</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	margin: 5px;
    background-color: orange;
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMais">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>380</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	margin: 5px;
    background-color: orange;
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>380</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	margin: 5px;
    background-color: rgb(85, 170, 255;
	background-color:);
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>380</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	margin: 5px;
    background-color: rgb(85, 170, 255;
	background-color:);
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>380</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	margin: 5px;
    background-color: rgb(85, 170, 255;
	background-color:);
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnIgual">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>470</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	margin: 5px;
    background-color: orange;
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVirgula">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>470</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	margin: 5px;
    background-color: rgb(85, 170, 255;
	background-color:);
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>,</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn0">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>470</y>
      <width>180</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	margin: 5px;
    background-color: rgb(85, 170, 255;
	background-color:);
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVezes">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	margin: 5px;
    background-color: orange;
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn8">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	margin: 5px;
    background-color: rgb(85, 170, 255;
	background-color:);
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn9">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	margin: 5px;
    background-color: rgb(85, 170, 255;
	background-color:);
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	margin: 5px;
    background-color: rgb(85, 170, 255;
	background-color:);
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDivisao">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	margin: 5px;
    background-color: orange;
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>÷</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMaismenos">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	margin: 5px;
    background-color: gray;
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPorcentagem">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	margin: 5px;
    background-color: gray;
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLimpar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	margin: 5px;
    background-color: gray;
    color: white;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QLabel" name="entrada">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>361</width>
      <height>111</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>420</width>
      <height>420</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>35</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="styleSheet">
     <string notr="true">	margin: 5px;
    background-color: white;
    color: black;
    padding: 5px;
    height: 20px;
    width: 20px;
    border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
